orchagent/switchorch.cpp

```diff
@@ -42,38 +46,67 @@ void SwitchOrch::doTask(Consumer &consumer)
         {
             for (auto i : kfvFieldsValues(t))
             {
-                auto attribute = fvField(i);
+                vector<string> attributes;
 
-                if (switch_attribute_map.find(attribute) == switch_attribute_map.end())
+                if (fvField(i) == "hash_seed")
```

Comment:
i wonder if we don't need such logics here. if we would like to set the two values with the same value, it is okay that we feed the `swssconfig` with the JSON file that contains both attributes with the same value. Thus the `for` loop below is not necessary.

Reply:
I agree that it would be better and logical to set hash seed attributes separately. I will update the pull request with appropriate change.
Since changes were done according to requirements document, I believe it should  be updated as well. (https://github.com/Azure/SONiC/wiki/ECMP-and-LAG-Hash-Seed)